67-fibonnacci-series.md

```diff
@@ -0,0 +1,52 @@
+# Write a program to produce Fibonacci series in Python
+
+`What is Fibonacci Series?` - It is a series of numbers formed by the sum of two  the two preceding numbers in the series.
+
+```python
+ 0,1,1,2,3,5
+```
+
+* The `1` is found by sum the two numbers before it (0+1)
+
+* The `2` is found by sum the two numbers before it (1+1)
+
+* The `3` is found by sum the two numbers before it (2+1)
+
+* The `5` is found by sum the two numbers before it (2+3)
+
+`It is done until the number of terms you want or requested by the user`.
+
+`Example:`
+
+```python
+e = int(input("Enter length: "))
+#first element of series
+l = 0  
+#second element of series                                      
+i=1                                       
+if e<=0:
+    print("The requested series is",l)
+else:
+    print(l,i,end=" ")
+    for x in range(2,e):
+        next=l+i                           
+        print(next,end=" ")
+        l = i
+        i = next
```

Comment:
I recommend to change those i, l and etc. to full names, it is hard to read.

Looks like formatting is messed up too, I recommend to use VS Code formatting functionality with shortcut: Shift+Alt+F, see full list: https://code.visualstudio.com/shortcuts/keyboard-shortcuts-windows.pdf